app/models/form/setup/pages/rent_type.rb

```diff
@@ -3,7 +3,6 @@ def initialize(_id, hsh, subsection)
     super("rent_type", hsh, subsection)
     @header = ""
     @description = ""
-    @depends_on = [{ "supported_housing_schemes_enabled?" => true }]
```

Comment:
This was incorrect and shouldn't have been here since rent_type is not supported housing specific.